internal/kafka/internal/config/aws.go

```diff
@@ -57,5 +72,16 @@ func (c *AWSConfig) ReadFiles() error {
 		return err
 	}
 	err = shared.ReadFileValueString(c.Route53SecretAccessKeyFile, &c.Route53SecretAccessKey)
+	if err != nil {
+		return err
+	}
+	err = shared.ReadFileValueString(c.SecretManagerAccessKeyFile, &c.SecretManagerAccessKey)
+	if err != nil {
+		return err
+	}
+	err = shared.ReadFileValueString(c.SecretManagerSecretAccessKeyFile, &c.SecretManagerSecretAccessKey)
+	if err != nil {
+		return err
+	}
 	return err
```

Comment:
unrelated to this PR, but here err is always `nil`. It would be more readable to return `nil`

Reply:
I'll do so.

Reply:
done, thanks!